src/app/features/observations/models.ts

```diff
@@ -21,6 +22,20 @@ export interface ObservationData {
   remarks: string;
 }
 
+export interface ObservationFilters {
```

Comment:
in order to be less verbose, you can create
```

export type DataGridFiltersObj<TData> = {
  [key in keyof TData]?: DataGridFilter[]
};
```

and extend this interface from DataGridFiltersObj<ObservationData>, leaving only overridden fields:
```


export interface ObservationFilters
  extends DataGridFiltersObj<ObservationData> {
  verificationStatus?: (DataGridFilter & { value: boolean })[];
  finder?: (DataGridFilter & { value: UserInfo })[];
}
```